deployments/get/objects.go

```diff
@@ -39,12 +64,11 @@ func GetClusterRoleBinding(namespace string) *rbacv1.ClusterRoleBinding {
 		},
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      clusterRoleBindingName,
-			Namespace: namespace,
```

Comment:
Was namespace redundant here?

Reply:
It's a `clusterrolebinding` which is applied across the cluster, so thought of removing this namespace field from this one. Tested this on GKE, nothing is breaking, so far. 
Having said that, leaving the namespace field as it as is also fine.